### ISSUE TITLE
Plugin CombineSites is not working!

### ISSUE THREAD

**Issue by guymass** [WWBN/AVideo#4028]
I wanted to test and try this plugin and connect it to another site and I get this error:

![Screen Shot 5781-03-02 at 19 50 07](https://user-images.githubusercontent.com/58882903/99568107-b4070b00-29d7-11eb-851f-55ee823025e5.jpg)

**Comment by akhilleusuggo** [comment]
both sites need to approve each other .

**Comment by JoshWho** [comment]
what does this plugin do? Never heard of it

**Comment by akhilleusuggo** [comment]
Actually this plugin is quite old . Maybe 3-4 months implemented , and if I'm not wrong was implemented with WWBN 

![image](https://user-images.githubusercontent.com/52116577/99627049-37f2de80-2a3c-11eb-8528-81919a2a6a21.png)

No idea what this plugin stands for . Does index the site , but for what ?

> Very curious about this, seems fascinating and if it is like that, than I can see a whole new way for using this script now. WE could be a huge network with millions of videos. I like this.

This will kill the SEO of all the sites , only 1 of them will get it basically . All the urls/videos gonna be tagged as "Duplicate without user-selected canonical" and "Duplicate, submitted URL not selected as canonical" in all search results . Gonna impact negatively .

I would use this plugin for 1 reason :

Main site too heavy , create a subdomain ( ww1 ) as a secondary site to handle the main streamer server and handle requests . 
Basically use it as a separated server working as a proxy/load balancing for main site ( database/cache )

**Comment by JoshWho** [comment]
Honestly if the dup was that bad of a thing then it wouldn't be copied with yt and other mirror sites as well. Seems like rel tag for origin is all that is needed for it not to be bad seo practice.

**Comment by akhilleusuggo** [comment]
Well , if you copy of any video from YouTube , your site will never be on the top or even on the first pages unless that video is deleted from YouTube ( not original content ) .

And yes , the title/description/tags plays a big roll . But are you gonna change all the titles of the sites you wanna combine ?
First you can't as far as I'm aware . 
Used it on home server for testing time ago , so no idea is has been any update .

**Comment by JoshWho** [comment]
Correct, not trying to out rank unless that is the goal then you would change the title but if you are trying to be a index resource it is nice to have access to a lot of info. Videos in this case.

**Comment by JoshWho** [comment]
Im seeing this as having a database of videos. It doesn't even have to be a rank issue for the extra videos it is just having them that makes your site better because people didn't have to go somewhere else to see it.

**Comment by JoshWho** [comment]
Would be kewl to make scheduled programs out of all this and make categories become watchable channels with tv guide.

**Comment by JoshWho** [comment]
This would be a global tv network that everyone can have a part in it it. I think it is brilliant if it works this way one day.

**Comment by akhilleusuggo** [comment]
Following with what you said , it's like using the API YouTube to embbed the entire YouTube xD . You can do that ofc .

**Comment by guymass** [comment]
Thanks guys for the answers, It sound like a good idea. I am using it because one site is overloaded so I figured maybe the second site can also display videos from the first, but currently I cannot see any videos listed from the main site to the secondary.

**Comment by akhilleusuggo** [comment]
do you have API plugin enabled ?

**Comment by guymass** [comment]
No I did not have it enabled, should it be?

**Comment by akhilleusuggo** [comment]
I think i works via API , so maybe you should

**Comment by guymass** [comment]
I am reopening this issue as I have combined my two sites and am getting an access error when trying to view videos from the other site on each of the site, even though the videos are shown. I am attaching some screen shots and as you can see I can list videos from either site on the front page of each site, but trying to open a video will give access denied.

![Uploading Screen Shot 5781-05-29 at 0.49.53.jpg…]()
![Uploading Screen Shot 5781-05-29 at 0.50.00.jpg…]()



![Screen Shot 5781-05-29 at 0 49 31](https://user-images.githubusercontent.com/58882903/107583158-341dc500-6c03-11eb-92d3-7b0835cb2756.jpg)
